### PR TITLE
Allow .js and .css

### PR DESCRIPTION
**Pull request by djbuch**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Branch?       | develop
| Description?  | In order to optimise "Mobile first" indexation
| Type?         | improvement
| Category?     | CO
| BC breaks?    | no
| Deprecations? | no
| Fixed ticket? | 
| How to test?  | Robot.txt must be updated with themes & modules informations.

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/12900)

### CODE REVIEW

File: classes/Tools.php [hunk 1]
```diff
@@ -2836,7 +2836,16 @@ public static function getRobotsContent()
             '*/modules/*.js',
             '*/modules/*.png',
             '*/modules/*.jpg',
```
Comment (PierreRambaud) [inline classes/Tools.php:2838]: So, can you add .gif, woff, woff2 & ttf in modules too.

Comment (djbuch) [reply classes/Tools.php:2838]: Yes, I've changed this

**Review by PierreRambaud — DISMISSED** [review]

**Comment by ntiepresta** [comment]
@djbuch 
The syntax error  is displayed when we testing this Pr, 
12900error

Best regards,
Reddy Ntie

File: classes/Tools.php [hunk 2]
```diff
@@ -2836,7 +2836,20 @@ public static function getRobotsContent()
...
-            '/js/jquery/*',
+            '*/modules/*.gif',
+            '*/modules/*.woff',
+            '*/modules/*.woff2',
+            '*/modules/*.ttf'
+            '*/js/*.css',
```
Comment (MathiasReker) [inline classes/Tools.php]:
```suggestion
            '*/js/*.js',
```

File: classes/Tools.php [hunk 3]
```diff
@@ -2836,7 +2836,20 @@ public static function getRobotsContent()
...
+            '*/js/*.js',
+            '*/themes/*.js',
+            '*/themes/*.css',
+            '*/themes/*.jpg',
+            '*/themes/*.png',
+            '*/themes/*.gif',
+            '*/themes/*.woff',
+            '*/themes/*.woff2',
+            '*/themes/*.ttf'
```
Comment (MathiasReker) [inline classes/Tools.php]:
```suggestion
            '*/themes/*.ttf',
```

**Review by PierreRambaud — APPROVED** [review]

File: classes/Tools.php [hunk 4]
```diff
@@ -2836,7 +2836,20 @@ public static function getRobotsContent()
             '*/modules/*.js',
             '*/modules/*.png',
             '*/modules/*.jpg',
-            '/js/jquery/*',
+            '*/modules/*.gif',
+            '*/modules/*.woff',
+            '*/modules/*.woff2',
+            '*/modules/*.ttf'
```
Comment (PierreRambaud) [inline classes/Tools.php:2842]: missing `,` here

**Review by PierreRambaud — CHANGES_REQUESTED** [review]
See my comment ;)

File: classes/Tools.php (hunk 4 above)
Comment (PierreRambaud) [inline classes/Tools.php:2842]:
```suggestion
            '*/modules/*.ttf',
```

**Comment by marionf** [comment]
Hello @djbuch 
Since we have not had news for more than 30 days, I close this PR
Feel free to open a new one if needed

**Comment by MathiasReker** [comment]
I can confirm this bug. This PR should be finished and merged.

**Comment by MathiasReker** [comment]
Use this tool to test it: https://technicalseo.com/tools/robots-txt/

**Comment by marionf** [comment]
> This PR should be finished and merged.

@MathiasReker Are you ok to finish it ?

**Comment by MathiasReker** [comment]
I cannot edit this PR. Just apply @PierreRambaud suggestion.

**Comment by marionf** [comment]
All right, I reopen it if you can apply the suggestions

File: classes/Tools.php (hunk 4 above)
Comment (MathiasReker) [reply classes/Tools.php:2842]: @marionf push this suggestion and it is OK to me :)

**Comment by PierreRambaud** [comment]
Hi @djbuch ,

Since we had no news from you for more than 30 days, I'll close this pull request. Feel free to reopen or open another one if you think it's still relevant.

Thanks!

**Pull request closed without merging**